src/odemis/gui/comp/text.py

```diff
@@ -932,7 +974,9 @@ def __init__(self, *args, **kwargs):
         kwargs['validator'] = IntegerValidator(min_val, max_val, choices, unit)
 
         if 'key_step' not in kwargs and (min_val != max_val):
```

Comment:
If the user passed `key_step_min`, you don't want to override it, so it should be:
`if 'key_step' not in kwargs and 'key_step_min' not in kwargs and (min_val != max_val):`

Reply:
Mhm the user should not pass key_step_min I thought. It should only be calculated if key_step was not provided. That is why I made this new key...

Reply:
ignore my comment